docusaurus.config.js

```diff
@@ -99,17 +99,24 @@ const config = {
               type: "docsVersionDropdown",
               position: "right",
               dropdownActiveClassDisabled: true,
+              dropdownItemsAfter: [
+                {
+                  href: "https://docs.cosmos.network/v0.46/",
+                  label: "v0.46",
+                  target: "_self",
+                },
+                {
+                  href: "https://docs.cosmos.network/v0.45/",
+                  label: "v0.45",
+                  target: "_self",
+                },
+              ],
             },
           ],
         },
         footer: {
           links: [
             {
-              items: [
```

Comment:
Why deleting the logo? IMHO it should stay and be copied under `static/img` to be properly resolved.

Reply:
`static/img/logo-bw.svg` there is still an issue too

Reply:
Ah, yes this is because it is getting it `https://cosmos.github.io/img/logo-bw.svg`, it will work just fine once this is in using a domain.